src/event-mixin.ts

```diff
@@ -37,7 +72,7 @@ export function AddEvents<TBase extends Constructor, U>(Base: TBase) {
     on<K extends keyof U, E extends eventHandlerType<U[K]>>(eventName: K, handler: E) {
       // Even though we bypass type safety in the call (casting this as any), we've enforced it in the
       // method signature above, so it's still safe.
```

Comment:
This comment looks like it isn't really relevant here anymore...maybe better as a note on `getEvent`?

Reply:
Good catch. Done.